js/test/backstop/backstopConfig.js

```diff
@@ -173,7 +179,8 @@ module.exports = ({ host, port }) => ({
       '--no-sandbox',
     ],
   },
-  asyncCaptureLimit: 5,
+  misMatchThreshold: 0,
```

Comment:
reduce to 0 as previous `View cookie[ preference]s` change hadn't been picked up